ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json

```diff
@@ -17,5 +17,25 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "respondent1ResponseCui",
```

Comment:
I am not sure that this is a good name for a field. i dont think ccd should know which project/application is populating this field. In this case this is a lip response. Perhaps it should be respondent1LipResponse?

Reply:
The structure of the model was @kenneth-hmcts 's idea. Regarding this he commented me that we should reference that the fields come from CUI.

Reply:
I agree with Tanya - model should always be separate from the view impl

Reply:
Solve, please review it again <3